src/dom.ts

```diff
@@ -332,12 +332,15 @@ export class ElementHandle<T extends Node = Node> extends js.JSHandle<T> {
     this._page._log(inputLog, `elementHandle.selectOption(%s)`, values);
     const deadline = this._page._timeoutSettings.computeDeadline(options);
     let vals: string[] | ElementHandle[] | types.SelectOption[];
-    if (!Array.isArray(values))
+    if (!values)
```

Comment:
- Let's do a strict check `values === null`.
- Please change the method and api.md to accept null: `selectOption(values: null | string | ...`